### ISSUE TITLE
roundup 2.1.0 build fails

### ISSUE THREAD

**Issue by deutrino** [turnkeylinux/tracker#1654]
Hello there, I tried bumping the version to 2.1.0 and building this appliance and the build is failing.

It looks like the build script expects a tree of files to end up at ```/home/roundup/.local/share``` which is not making it there.

I found that some of the included docs instead ended up at ```home/roundup/.local/lib/python3.7/site-packages/usr/share/doc/roundup/``` which may be a clue.

Unfortunately I'm a bit out of my depth at the moment as to what exactly is going on, so I'll paste my log. Note that I made one small change to the build script while attempting to get it to build, namely I added the line ```mkdir -p /home/roundup/.local/share``` because that directory didn't even exist and the build was originally blowing up there. The log below gets a little further before it fails.

As noted before, the directory it tried to access, in this case ```/home/roundup/.local/share/doc/roundup/html/_images```, ended up in ```home/roundup/.local/lib/python3.7/site-packages/usr/share/doc/roundup/html/_images/```

```
$(call run-conf-scripts, conf.d)
fab-chroot build/root.patched --script conf.d/main
+ NAME=roundup
+ USER=roundup
+ ADMIN_PASS=turnkey
+ mcookie
+ DB_PASS=786a0643d5d942a51db6336b2e2652d1
+ RT_TRACKER=/var/lib/roundup/tracker
+ RT_TEMPLATE=classic
+ RT_BACKEND=mysql
+ RT_VER=2.1.0
+ RT_CONF=/etc/roundup/tracker-config.ini
+ RT_LOCAL=/home/roundup/.local
+ python3 --version+ sed s|^.* \(3\.[0-9]\+\).*|\1|

+ PY_V=3.7
+ adduser --disabled-login --gecos roundup user roundup
Adding user `roundup' ...
Adding new group `roundup' (1000) ...
Adding new user `roundup' (1000) with group `roundup' ...
Creating home directory `/home/roundup' ...
Copying files from `/etc/skel' ...
+ su - roundup -c pip3 install 'roundup==2.1.0' --user
Collecting roundup==2.1.0
  Downloading https://files.pythonhosted.org/packages/80/6d/3ccdd66551bb89b90758add63b8814b516d7471348675aa564aab032341f/roundup-2.1.0.tar.gz (3.6MB)
    100% |████████████████████████████████| 3.6MB 382kB/s
Building wheels for collected packages: roundup
  Running setup.py bdist_wheel for roundup ... done
  Stored in directory: /home/roundup/.cache/pip/wheels/96/6b/4f/82e98d6a6fc6b28e7a8fc98e4398ba80e2bbfed3dee3de5054
Successfully built roundup
Installing collected packages: roundup
  The scripts roundup-admin, roundup-demo, roundup-gettext, roundup-mailgw, roundup-server and roundup-xmlrpc-server are installed in '/home/roundup/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed roundup-2.1.0
+ mkdir -p /home/roundup/.local/share
+ chmod -R 0775 /home/roundup/.local
+ chmod -R 0774 /home/roundup/.local/share
+ DOCS=/home/roundup/.local/share/doc/roundup/html
+ SUB_DIRS=_images _sources _static
+ chmod o+x /home/roundup/.local/share/doc/roundup/html/_images
chmod: cannot access '/home/roundup/.local/share/doc/roundup/html/_images': No such file or directory
make: *** [/usr/share/fab/product.mk:540: build/stamps/root.patched] Error 1
```

**Comment by deutrino** [comment]
I've made a little progress trying to get the build to complete, though now it's stopping for user input at a point further along:

```
+ mkdir -p /var/lib/roundup/tracker
+ mkdir /etc/roundup
+ chown -R roundup:roundup /var/lib/roundup/tracker
+ OPTIONS=admin_email=admin,dispatcher_email=admin,tracker_web=https://www.example.com/,mail_domain=example.com,mail_host=localhost,rdbms_password=63dd9066fc900bafd569906414ebfe65
+ su - roundup -c roundup-admin -i /var/lib/roundup/tracker install classic mysql admin_email=admin,dispatcher_email=admin,tracker_web=https://www.example.com/,mail_domain=example.com,mail_host=localhost,rdbms_password=63dd9066fc900bafd569906414ebfe65
Templates:: 
Select template [classic]:
```

**Comment by JedMeister** [comment]
Hi @deutrino 

Following on from another user posting in the [forums](https://www.turnkeylinux.org/forum/support/sat-20211030-2241/roundup-tracker-how-import) about issues with the current Roundup appliance, I have had a bit of a dive into this.

I managed to update roundup on the current appliance and have [documented it in a forum post](https://www.turnkeylinux.org/forum/support/sat-20211030-2241/roundup-tracker-how-import#comment-50403). It's a little messy, but it appears to work ok.

If you are still wanting to rebuild that appliance, hopefully you can work out how to update the build code to use those new paths?! Or if you just want the new version of Roundup, you could just follow those instructions to update Roundup in the current appliance.

If you do update the appliance build code, please feel free to open a pull request! :wink:

**Comment by JedMeister** [comment]
Fixed in v17.0
